packages/data-point/test/utils/reducers.js

```diff
@@ -16,88 +16,78 @@ module.exports.invalidReducer = () => {
 }
 
 // invalid in arity, must be 2
-module.exports.invalidReducerArity = () => context => {
-  // return method should have context and callback as arguments
+module.exports.invalidReducerArity = () => value => {
+  // return method should have acc and callback as arguments
 }
 
-module.exports.isEqualTo = value => (context, next) => {
-  next(null, context.value === value)
+module.exports.isEqualTo = compareTo => value => {
+  return compareTo === value
 }
 
-module.exports.addKeyValue = (key, val) => (context, next) => {
-  const value = _.set(context.value, key, val)
-  next(null, value)
+module.exports.addKeyValue = (key, val) => value => {
+  // TODO: check why this is mutating object, dont believe it should
```

Comment:
is this something we need to address? set does modify the object...

Reply:
yeah, not part of this PR but yeah when I was refactoring found that and a couple more, reason why I left that note